.claude/hooks/stop.py

```
@@ -0,0 +1,171 @@
+
+        # Extract required fields
+        session_id = input_data.get("session_id", "")
+        stop_hook_active = input_data.get("stop_hook_active", False)
+
+        # Ensure session log directory exists
+        log_dir = ensure_session_log_dir(session_id)
+        log_path = log_dir / "stop.json"
+
+        # Read existing log data or initialize empty list
+        if log_path.exists():
+            with open(log_path, "r") as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)